web/templates/settings/notification_settings.hbs

```diff
@@ -107,6 +107,8 @@
             {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings) }}
         </div>
 
+        <div class="settings_banners_wrapper"></div>
```

Comment:
This name is not unique enough; also all new classes should use dashes. Changed to `desktop-notification-settings-banners`.